Linux-Course/README.md

```diff
@@ -0,0 +1,157 @@
+# [Sudoer.ir](https://www.sudoer.ir "Former LLinux") Linux Course 
+
+## INTRO
+
+- [Intro 1](https://sudoer.ir/video/linux/sudoer_001.mp4 "Intro 1")
```

Comment:
Hi thank you so much
Please mention also youtube link beside sudoer.ir links youtube.com/@sudoergrp

Reply:
Thanks